tools/slack/guidelines.md

```diff
@@ -76,6 +76,31 @@ Slack Groups allow you to direct messages to a specific list of people in a more
 
 Groups are a great way to alert people who might not be in a channel about something that needs their attention or make sure urgent incidents are directed to the right people and not an entire channel. For example, you can ping **@github-admins** in #admins-github if you have an urgent issue instead of using an **@channel** or use **@federalist-team** to alert that group of a conversation in another channel that we should see immediately.
 
+## Custom emoji
+
+Emoji are an important part of our communication within Slack. You are welcome
+to add custom emoji. Keep in mind that interactions in Slack are covered by our
+[Code of Conduct]({% link _pages/policies/conduct-policies/code-of-conduct.md %}), including emoji.
+
+To create custom emoji:
+
+1. Open [#emoji-showcase](https://gsa-tts.slack.com/archives/C0X2T36AY) in Slack
+1. Use the Shortcuts button (lightning bolt) and select "Request new emoji"
+1. Fill out the form (name and URL) and click submit
+
+![Request new emoji form]({{site.baseurl}}/images/slack-and-social-media/request-new-emoji-form.png)
```

Comment:
```suggestion
```

Reply:
I see what you're doing here. Is there a content design best practice here for images? I feel like the image is a helpful visual and I'd prefer not to hide it behind a link.